lib/omniauth/strategies/bigcommerce.rb

```diff
@@ -51,7 +51,8 @@ class BigCommerce < OmniAuth::Strategies::OAuth2
         {
           raw_info: raw_info,
           scopes: raw_info['scope'],
-          context: raw_info['context']
+          context: raw_info['context'],
+          account_uuid: raw_info['account_uuid']
```

Comment:
`raw_info.fetch('account_uuid', '')` since we don't have guarantees this will be there